asserts/database_test.go

```diff
@@ -655,7 +655,6 @@ func (safs *signAddFindSuite) TestSignDelegation(c *C) {
 
 	// now add authority-delegation
 	headers = map[string]interface{}{
-		"type":         "account-key",
```

Comment:
These changes got me very confused, but I noticed there is a separate commit for them and the commit message clarifies it, so 👍